### PR TITLE
DOC: Ingress setup doc

### PR DESCRIPTION
**Pull request by Shubham9t9**
# Description

Documentation for ingress setup for devtron installation

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# Checklist:

* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [X] Does this PR require documentation updates?
* [X] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have tested it for all user roles

### CODE REVIEW

File: docs/setup/install/ingress-setup.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              number: 80
+        path: /orchestrator
+        pathType: ImplementationSpecific 
+      - backend:
+          service:
+            name: devtron-service
+            port:
+              number: 80
+        path: /dashboard
+        pathType: ImplementationSpecific
```
Comment (prakarsh-dt) [inline docs/setup/install/ingress-setup.md:39]: Update the ingress yaml, the updated one has an entry for /grafana too

File: docs/setup/install/ingress-setup.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  - http:
+      paths:
+      - backend:
+          serviceName: devtron-service
+          servicePort: 80
+        path: /orchestrator
+      - backend:
+          serviceName: devtron-service
+          servicePort: 80
+        path: /dashboard
```
Comment (prakarsh-dt) [inline docs/setup/install/ingress-setup.md:63]: add /grafana as per updated ingress

File: docs/setup/install/ingress-setup.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        path: /orchestrator
+        pathType: ImplementationSpecific 
+      - backend:
+          service:
+            name: devtron-service
+            port:
+              number: 80
+       Host: devtron.example.com
+        path: /dashboard
+        pathType: ImplementationSpecific
```
Comment (prakarsh-dt) [inline docs/setup/install/ingress-setup.md:96]: add /grafana

File: docs/setup/install/ingress-setup.md (hunk 1 above)
Comment (Shubham9t9) [reply docs/setup/install/ingress-setup.md:39]: updated

File: docs/setup/install/ingress-setup.md (hunk 3 above)
Comment (Shubham9t9) [reply docs/setup/install/ingress-setup.md:110]: added

File: docs/setup/install/ingress-setup.md (hunk 2 above)
Comment (Shubham9t9) [reply docs/setup/install/ingress-setup.md:70]: added

**Review by prakarsh-dt — DISMISSED** [review]

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**